new-components/pkg/test/util/test_transformer_wrapper.go

```diff
@@ -0,0 +1,202 @@
+package testutil
```

Comment:
Sorry to be pedantic here, can we keep this in the needed component for now?

I'd like us to keep implementing the same bit a few times before it needs to be abstracted in another module